Documentation/trying-out-rkt.md

```diff
@@ -16,6 +16,8 @@ cd rkt-v1.8.0
 ./rkt help
 ```
 
+Note that in order to fully utilize rkt there is a couple of additional steps to be performed. Therefore we highly recommend installing rkt using your local package manager. If rkt is not available for your Linux distribution please consider to try rkt using Vagrant as described below.
```

Comment:
Agreed, the whole paragraph actually gives a too simplified view to get started on Linux, since there are quite a few directories to be created, stage1 images to be copied, and potentially other operations (manpages, bash completion, etc.).

Maybe we just recommend a VM-only usage based on Vagrant, or installing rkt via a package manager and remove this paragraph completely?

@joshix thoughts?